src/applications/vaos/tests/e2e/page-objects/AppointmentList/AppointmentDetailPageObject.js

```diff
@@ -106,18 +105,14 @@ class AppointmentDetailPageObject extends PageObject {
       assertLocation(type, tokens);
 
       // And the start time should match the appointment
-      expect(tokens.get('DTSTAMP')).to.equal(
-        formatInTimeZone(startDate, 'UTC', DATE_FORMATS.iCalDateTimeUTC),
+      expect(tokens.get('DTSTAMP')).to.contain(
+        format(startDate, DATE_FORMATS.iCalDateTime),
```

Comment:
This seems to be the wrong behavior since `generateICS` is always formatting the date in UTC time and we want to ensure the test asserts that exactly. I looked through the code and I think the issue has to do with how start times are passed into the `MockAppointmentResponse` constructor. For example, start is formatted using the UTC string but should be formatted using the 'UTC' timezone (i.e. `formatInTimezone(timestamp, 'UTC', DATE_FORMATS.ISODateTimeUTC)`). There are several instances in that constructor where I think we should be using `formatInTimezone` instead as well but I'll leave it up to you to decide which actually needs to be included in this PR.

Let me know if this makes sense at all.